### ISSUE TITLE
Why 112x112?

### ISSUE THREAD

**Issue by yaseryacoob** [mapooon/Face2Diffusion#4]
In reading the paper and the repo I am wondering why you chose 112x112 as the face resolution? Given other embeddings are in the 200+ and given that one could go to 512x512 to capture Identity even better. 

thanks for sharing your code.

**Comment by mapooon** [comment]
Thank you for your interest in our work.
This is because the face recognition (identity encoding) part of our code is based on [insightface](https://github.com/deepinsight/insightface/tree/master/recognition/arcface_torch) that uses 112x112 images.
In practice, 112x112 images work well in recognizing (identifying) faces.

**Comment by yaseryacoob** [comment]
The issue is that face recognition type embeddings are (or maybe) good for face recognition, but more is needed for generative type purposes. This is why you had to have the expression encoding. I wonder if a richer embedding can be computed. It is an open question, beyond your work. Thanks for sharing your work!

**Comment by mapooon** [comment]
Exactly. 112x112 images are considered sufficient for face recognition, but not obvious for generative purposes. Higher resolution may yield better results for generative models as you mentioned.

**Comment by gaoyixuan111** [comment]
Thank you for your excellent work. Could you please share your training process readme and training time，datasets?

**Comment by Andyplus1** [comment]
> Thank you for your excellent work. Could you please share your training process readme and training time，datasets?

请问您有得到相关代码吗
